<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentMethod</class>
 <widget class="QMainWindow" name="PaymentMethod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(136, 202, 94);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>430</y>
      <width>811</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(136, 202, 94);
font: 75 10pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Â© 2024 LMMN, Inc. All rights reserved</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="CashButton">
     <property name="geometry">
      <rect>
       <x>-70</x>
       <y>-100</y>
       <width>391</width>
       <height>411</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Resources/images/method_pesos.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 19pt &quot;MS Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Choose a payment method</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Sans Serif&quot;;
border-radius: 10px; 
background-color: rgb(255, 255, 255);
color: rgb(136, 202, 94);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>(Select one of the options below)</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="EcashButton">
     <property name="geometry">
      <rect>
       <x>-50</x>
       <y>-70</y>
       <width>351</width>
       <height>351</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Resources/images/method_ecash.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Pay with cash (coins and bills)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Pay with E-cash</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Click the respective icons  for each method to proceed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PayCashButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>321</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0px;
border-thickness: 0px;
border-radius:20px;
</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PayeCashButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>321</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:0px;
border-thickness: 0px;
border-radius:20px;
</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <zorder>PayeCashButton</zorder>
   <zorder>PayCashButton</zorder>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
   <zorder>label_2</zorder>
   <zorder>back_Button</zorder>
   <zorder>label_3</zorder>
   <zorder>frame_3</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
